SpatialGDK/Source/SpatialGDK/Private/Tests/SpatialView/ExpectedViewDelta.cpp

```diff
@@ -104,14 +108,34 @@ bool ExpectedViewDelta::Compare(const ViewDelta& Other)
 			return false;
 		}
 
-		if (LhsEntityDelta.bAdded != RhsEntityDelta.bAdded)
+		switch (LhsEntityDelta.Type)
```

Comment:
Can this whole thing be replaced by
```
if (LhsEntityDelta.Type != RhsEntityDelta.Type)
{
    return false;
}
```

Reply:
I think that depends on the ordering of the respective enums in `EntityDelta` and `ExpectedEntityDelta`

Reply:
Actually can you even compare two different enums?

Reply:
could we just use the same enum for both `EntityDelta` and `ExpectedEntityDelta`?

Reply:
Oh they're different enums. If the enum isn't anonymous then yes. Otherwise you can give them explicit integer values, but I agree I'd rather not rely on us getting that right.

Reply:
If this is the only code this is going to make nicer then I don't care

Reply:
this is the only place where we compare these